migrations/helpers/update_filtered_positives_helper.py

```diff
@@ -0,0 +1,270 @@
+
+    Arguments:
+        config {ModuleType} -- application config specifying database details
+        samples {List[Dict[str, str]]} -- the list of samples whose filtered positive fields
+        should be updated
+
+    Returns:
+        bool -- whether the updates completed successfully
+    """
+    mysql_conn = create_mysql_connection(config, False)
+
+    if mysql_conn is not None and mysql_conn.is_connected():
+        mlwh_samples = [map_mongo_to_sql_common(sample) for sample in samples]
+        run_mysql_executemany_query(
+            mysql_conn, SQL_MLWH_MULTIPLE_FILTERED_POSITIVE_UPDATE, mlwh_samples
+        )
+        return True
+    else:
+        return False
+
+
+def update_dart_filtered_positive_fields(config: ModuleType, samples: List[Dict[str, Any]]) -> bool:
+    """Updates DART plates and wells following updates to the filtered positive fields
+
+    Arguments:
+        config {ModuleType} -- application config specifying database details
+        samples {List[Dict[str, str]]} -- the list of samples to update in DART
+
+    Returns:
+        bool -- whether the updates completed successfully
+    """
+    sql_server_connection = create_dart_sql_server_conn(config, False)
+    if sql_server_connection is None:
+        raise ValueError("Unable to establish DART SQL Server connection")
+
+    dart_updated_successfully = True
+    centres = config.CENTRES  # type: ignore
+    labclass_by_centre_name = biomek_labclass_by_centre_name(centres)
+    try:
+        cursor = sql_server_connection.cursor()
+
+        for plate_barcode, samples_in_plate in groupby_transform(
+            samples, lambda x: x[FIELD_PLATE_BARCODE], reducefunc=lambda x: list(x)
+        ):
+            try:
+                labware_class = labclass_by_centre_name[samples_in_plate[0][FIELD_SOURCE]]
+                plate_state = add_dart_plate_if_doesnt_exist(
+                    cursor, plate_barcode, labware_class  # type:ignore
+                )
+                if plate_state == DART_STATE_PENDING:
+                    for sample in samples_in_plate:
+                        well_index = get_dart_well_index(sample.get(FIELD_COORDINATE, None))
+                        if well_index is not None:
+                            well_props = map_mongo_doc_to_dart_well_props(sample)
+                            set_dart_well_properties(
+                                cursor, plate_barcode, well_props, well_index  # type:ignore
+                            )
+                        else:
+                            raise ValueError(
+                                "Unable to determine DART well index for sample "
+                                f"{sample[FIELD_ROOT_SAMPLE_ID]} in plate {plate_barcode}"
+                            )
+                cursor.commit()
+                dart_updated_successfully &= True
+            except Exception:
+                print(f"** Failed updating DART for samples in plate {plate_barcode} **")
```

Comment:
I know I've read over the other print statements but I just thought about it now that we should actually be using the built-in logging functionality instead of bare printing to console. The logging functionality gives us much more flexibility.

Reply:
Logging added 😄